src/bundle/Resources/views/content/content_view_fields.html.twig

```diff
@@ -1,32 +1,44 @@
 <section class="ez-view-rawcontentview">
+    {% set preview_is_collapsed = ez_user_settings['collapse_content_preview'] %}
     <div class="ez-raw-content-title d-flex justify-content-between mb-3">
         <h2>
-            {{ 'tab.view.content'|trans()|desc('Content') }}
+            <a data-toggle="collapse" href=".ez-content-preview-collapse" {% if preview_is_collapsed %}class="collapsed"{% endif %}>
```

Comment:
Please add some class to this `a` element and style it by the class.

Reply:
Fixed https://github.com/ezsystems/ezplatform-admin-ui/pull/860/commits/74642645add985253a5b9050c2405818fc255b20